mobile_verifier/migrations/21_index_hex_coverage.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS hex_coverage_radio_key_idx ON hex_coverage USING btree (radio_key)
```

Comment:
Isn’t btree the default? Just looking for clarity?

Reply:
yes, I just copied the command that was generated by my sql client